src/Makefile

```diff
@@ -37,8 +37,16 @@ include $(TOP)/src/common.inc
 include $(TOP)/src/version.inc
 
 TARGETS = libpmem libvmem libpmemblk libpmemlog libpmemobj libpmempool\
-	libvmmalloc tools
-ALL_TARGETS = $(TARGETS) common librpmem examples benchmarks
+	libvmmalloc
+
+$(warning $(BUILD_AARCH64))
+ifneq ($(BUILD_AARCH64),y)
+	ALL_TARGETS = $(TARGETS) common librpmem examples benchmarks tools
+else
+	ALL_TARGETS = $(TARGETS) common
+endif
+$(warning $(ALL_TARGETS))
```

Comment:
Leftovers?

Reply:
This patch currently does not port everything, like tools or examples, benchmarks etc, but only those that are required to compile libpmemobj on ARM. So I have a added a way to skip the rest while building on ARM, this though will not affect regular compilation as user has to explicitly ask for ARM processor builds.

Reply:
What I actually meant was the `$(warning)` prints.